mod/github.go

```diff
@@ -190,3 +207,19 @@ func writeFile(filename string, content []byte) error {
 	}
 	return nil
 }
+
+const SHA_LENGTH = 12
+
+func (d *githubMgr) CacheRef(repoPath *githubRepoPath, ref string) (string, error) {
```

Comment:
A little bit confused by the function name here, from what I can see, this function takes a repopath and a ref which is either
- A tag
- A branch name
And returns a string to "uniquely identify" that set of files?

Reply:
yup